R/h_bracket.R

```diff
@@ -0,0 +1,21 @@
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "character",
+    definition = function(x) {
+       if(nchar(x) >= 1) paste0("{\n", paste0(x, collapse = "\n"), "\n}\n")
+    }
+)
+
+#' @rdname h_bracket
+#' @export
+setMethod(
+    f = "h_bracket",
+    signature = "list",
+    definition = function(x) {
+        model_text <- paste0(x, collapse = "\n")
+        paste0( model_text, "\n", "target+=sum(log_lik);\n")
+
+    }
+)
```

Comment:
Why does this need to be a S4 object and not just a  general function ?

Reply:
I want the function to treat different list inputs (which is the prior lists) and character strings ( that is all the rest slots of StanModule). Thus, I need it to be generic. I am using the S4 method ( instead of S3) in order to be similar with the rest of the package.

Reply:
urgh, yer fair enough. Personally I don't like S4 and try to use it as little as I can get away with but I appreciate thats purely a subjective argument and not based on anything realistic so am fine with this :)